bindata/scripts/bf2-switch-mode.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+set -e
+
+mode=${1:-query}
+device=${2:-"$(mstconfig q | grep "Device:" | awk 'BEGIN {FS="/";} { print $6 }')"}
```

Comment:
should you ensure the selected device is BF2 ?

Reply:
This is a good point, I would have added 

```
type=$(mstconfig q $device | grep "Device type:" | awk '{ print $3}')

if [ "${type}" != "BlueField2" ]; then
    echo "Device is not a Bluefield2"
    exit 120
fi
```